app/main.py

```diff
@@ -1,13 +1,24 @@
+from typing import Dict, Any
```

Comment:
The import statement has a syntax error. The correct syntax should be `from typing import Dict, Any` without the slash (`/`).